mmv1/third_party/terraform/services/logging/resource_logging_project_sink.go

```diff
@@ -175,6 +183,14 @@ func resourceLoggingProjectSinkUpdate(d *schema.ResourceData, meta interface{})
 }
 
 func resourceLoggingProjectSinkDelete(d *schema.ResourceData, meta interface{}) error {
+	name := d.Get("name")
+	for _, restrictedName := range []string{"_Required", "_Default"} {
+		if name == restrictedName {
+			log.Print("[WARN] Default logging sinks cannot be deleted.")
+			return nil
```

Comment:
I think we need to zero-out the resource's ID here, similar to https://github.com/GoogleCloudPlatform/magic-modules/pull/9384/files#diff-678adc4a8a3f48a67e1b37f50964412a80d5c9cb42e5a116a45489e1946774e3R205

Reply:
I'm following the same pattern that is in use for `google_logging_bucket_config` which does not zero-out the IDs:  https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/logging/resource_logging_bucket_config.go#L381

Shoud `google_logging_bucket_config` also be updated to zero it out?

Reply:
Nevermind, seems like this is redundant to do. Let's keep it the same as `google_logging_bucket_config`